packages/components/src/internal/components/editable/utils.ts

```diff
@@ -13,8 +13,11 @@ import { getUpdatedDataFromGrid } from '../../util/utils';
 
 import { EXPORT_TYPES } from '../../constants';
 
-import { EditableGridLoaderFromSelection } from './EditableGridLoaderFromSelection';
-
+/**
+ * @deprecated Use initEditableGridModel() or initEditableGridModels() instead.
+ * This method does not make use the grid loader paradigm. Usages of this method directly
+ * is susceptible to data misalignment errors with the associated view.
+ */
```

Comment:
are there very many usages of this? I see a couple in LKFM. Just trying to get an idea of how big of a refactor this would be so that we can just remove this after we convert those usages.

Reply:
This method is still in use within the file so we might not get rid of it but rather no longer export it. That said, the conversion requires updating the usages to use the `EditorModel` paradigm and initialization pattern. I didn't look too closely but it was more than I wanted to tackle with these changes.